src/tink/http/Header.hx

```diff
@@ -91,12 +91,23 @@ class Header {
     
   public function concat(fields:Array<HeaderField>):Header
     return new Header(this.fields.concat(fields));
+    
+  public function getContentLength()
+    return switch byName(CONTENT_LENGTH) {
+      case Success(Std.parseInt(_) => null): Failure(new Error(UnprocessableEntity, 'Invalid Content-Length Header'));
+      case Success(Std.parseInt(_) => v): Success(v);
+      case Failure(e) if(e.message == headerNotFound(CONTENT_LENGTH)): Success(0); // this should be ok
```

Comment:
I'm not sure it is. If the transfer encoding is chunked, then no content length is specified, but it's not 0 either.